recipe/thirdparty/tpmodels/recipeInterface.go

```diff
@@ -18,16 +18,22 @@ package tpmodels
 import "github.com/supertokens/supertokens-golang/supertokens"
 
 type RecipeInterface struct {
-	GetUserByID                *func(userID string, userContext supertokens.UserContext) (*User, error)
-	GetUsersByEmail            *func(email string, userContext supertokens.UserContext) ([]User, error)
-	GetUserByThirdPartyInfo    *func(thirdPartyID string, thirdPartyUserID string, userContext supertokens.UserContext) (*User, error)
+	GetUserByID             *func(userID string, userContext supertokens.UserContext) (*User, error)
+	GetUsersByEmail         *func(email string, userContext supertokens.UserContext) ([]User, error)
+	GetUserByThirdPartyInfo *func(thirdPartyID string, thirdPartyUserID string, tenantId *string, userContext supertokens.UserContext) (*User, error)
+
+	GetProvider *func(thirdPartyID string, tenantId *string, userContext supertokens.UserContext) (GetProviderResponse, error)
+
 	SignInUp                   *func(thirdPartyID string, thirdPartyUserID string, email string, oAuthTokens TypeOAuthTokens, rawUserInfoFromProvider TypeRawUserInfoFromProvider, tenantId *string, userContext supertokens.UserContext) (SignInUpResponse, error)
-	ManuallyCreateOrUpdateUser *func(thirdPartyID string, thirdPartyIserID string, email string, userContext supertokens.UserContext) (ManuallyCreateOrUpdateUserResponse, error)
+	ManuallyCreateOrUpdateUser *func(thirdPartyID string, thirdPartyUserID string, email string, tenantId *string, userContext supertokens.UserContext) (ManuallyCreateOrUpdateUserResponse, error)
+}
 
-	CreateOrUpdateThirdPartyConfig *func(thirdPartyId string, tenantId *string, config ProviderConfig, userContext supertokens.UserContext) (CreateOrUpdateTenantIdConfigResponse, error)
-	FetchThirdPartyConfig          *func(thirdPartyId string, tenantId *string, userContext supertokens.UserContext) (FetchTenantIdConfigResponse, error)
-	DeleteThirdPartyConfig         *func(thirdPartyId string, tenantId *string, userContext supertokens.UserContext) (DeleteTenantIdConfigResponse, error)
-	ListThirdPartyConfigs          *func(tenantId *string, userContext supertokens.UserContext) (ListTenantConfigMappingsResponse, error)
+type GetProviderResponse struct {
+	OK *struct {
+		Provider          TypeProvider
+		ThirdPartyEnabled bool
+	}
+	TenantDoesNotExistError *struct{}
```

Comment:
Do not add `TenantDoesNotExistError` to the interface

Reply:
Done